packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -84,6 +85,7 @@ interface IFocusable {
  */
 @customElement('mgt-people-picker')
 export class MgtPeoplePicker extends MgtTemplatedComponent {
+  private _uid: string;
```

Comment:
@gavinbarron how did the lack of unique element IDs for the component affect it with voice over? I need some insights to tackle #2128

Reply:
I'm not sure that it necessarily did have an impact on VoiceOver. But if we have two or more people-pickers in the same pages then we'd have duplicate ids and I'd rather not render invalid html if we can avoid it easily.

Reply:
I did some more research, because each shadow root is considered it's own document then we don't need to worry about the case when we have 2 or more instances of a control on a page as the "duplicate" ids will exist in different shadow roots.
I'll revert this part of the change